analysis_options.yaml

```diff
@@ -1,12 +1,14 @@
+include: package:workiva_analysis_options/v1.recommended.yaml
+
 analyzer:
-  strong-mode:
-    implicit-casts: true
-    implicit-dynamic: true
+  errors:
+    must_call_super: error
+    unused_import: warning
+    duplicate_import: warning
+    comment_references: info
+    prefer_void_to_null: info
```

Comment:
Why just an info here?

Reply:
Mainly because I was unsure if changing `Future<Null>` to `Future<void>` would be a breaking change...

I can try addressing them instead though, and seeing what semver audit says...

Reply:
Actually it looks like the ones I kept around have ignore comments on them since they appear to be `Null` on purpose for things like [arity checks](https://github.com/Workiva/react-dart/pull/197)... so I'll remove this downgrade.